### ISSUE TITLE
Map for the "Fourth Battle of Kharkov" scenario from the "DasReich" campaign

### ISSUE THREAD

**Issue by salexgit** [nicupavel/openpanzer#141]
Hi

It is not a bug, but rather a logical inconsistency in map files. The fourth battle of Kharkov took place in August 1943, while the corresponding scenario from the "Das Reich" campaign uses the winter map, which is really obscure.

Fortunatelly enough, there is exactly the same summer map (ID#1241).

![1241](https://user-images.githubusercontent.com/22852936/42193383-ba9ff53a-7e5d-11e8-9b83-52e08155bb3b.jpg)

So, the fix should be quite straightforward: modify the corresponding .SCN file so that it points to the right .MAP file.

**Comment by salexgit** [comment]
Hi again

The fix turned out a bit more complex because as of now only .MAPX file exists for map ID#1241 (which is a bit strange as in most cases both .MAPX and .MAP files are provided), whereas .SCN file needs .MAP format. So, what I did is that I created .MAP file from .MAPX and use it for the scenario file.

I am attaching here the corresponding files:
- KHARKOVS.map (converted from kharkov-summer.mapx)
- DRKHAR23.scn (which points now to KHARKOVS.map)
- drkhar23.xml (converted scenario file into the OpenPanzer format)

Developers just need to overwrite DRKHAR23.scn and add KHARKOVS.map file into the repository. 

[KHARKOVS.map.zip](https://github.com/nicupavel/openpanzer/files/2161371/KHARKOVS.map.zip)
[drkhar23.scn.zip](https://github.com/nicupavel/openpanzer/files/2161372/drkhar23.scn.zip)
[drkhar23.xml.zip](https://github.com/nicupavel/openpanzer/files/2161375/drkhar23.xml.zip)
